packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -112,6 +109,13 @@ const Dashboard = observer(() => {
                     inline: 'center',
                 });
             }
+
+            if (tab_index === BOT_BUILDER) {
+                RudderStack.track('ce_bot_builder_form', {
```

Comment:
isn't dashboard?

Reply:
@NikitK-deriv  this is to check the bot header form because we need to see if the user has landed on the bot builder page using navigation tab

Reply:
I see, thank. Let's keep it then